book/index.rst

```diff
@@ -1,5 +1,11 @@
 The Book
-========
+********
```

Comment:
Right, RST implies the priority of the first appearance, but the [Symfony Documentation standard](http://symfony.com/doc/current/contributing/documentation/standards.html) has some rules :-)